app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+
+<div>
+  <h1><%= @user.name.capitalize %>'s Events</h1>
+  <i class="fa fa-calender" aria-hidden="true"></i></span>
+  <% if @events_by_current_user %>
+    <h3>Created Events</h3>
+    <%= render @events_by_current_user %>
+
```

Comment:
- [ ] You can't use a render method, it expects a file path as an argument. I would suggest you loop through the array of the event object - @events_by_current_user, which is what the variable holds.